doc/run_on_minikube.md

```diff
@@ -0,0 +1,113 @@
+# Run PaddleCloud on your local machine
+
+This documentation shows how to run PaddleCloud on minikube.   
+(This surports only mac now.On unbuntu-16.04,we met a bug of kubernetes when we use `hostpath` to volume a file.)
+
+## Prerequisites
+
+- [install minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/)
+- [install kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+1. Start a local minikube cluster:
+
+    ```bash
+    minikube start --kubernetes-version v1.6.4
+    ```
+    
+    If you can't connect to minikube distribution server,add https_proxy like that:
+    
+    ```bash
+    https_proxy=<your's> minikube start --kubernetes-version v1.6.4
+    ```
+1. Enable ingress addon:
+
+ 	```
+ 	minikube addons enable ingress
+ 	```
+ 	
+1. Create workspace directory:
+
+	```
+	mkdir <yourpath>
+	```  
+	Minikube mount `$Home` path default,and you'd better to create path under it or you may find that it can't be changed in minikube virtual mathine or in kubertes pod.
+	
+1. Copy `ca` and generate `admin` certificate：    
+	(We must use `ca` under `~/.minikube` instead of under `~/.minikube/certs`.)
+	
+	```
+	mkdir <yourpath>/certs && cd <yourpath>/certs
+	openssl genrsa -out admin-key.pem 2048
+	openssl req -new -key admin-key.pem -out admin.csr -subj "/CN=kube-admin"
+	openssl x509 -req -in admin.csr -CA ~/.minikube/ca.crt -CAkey ~/.minikube/ca.key \
+  		-CAcreateserial -out admin.pem -days 365
+	cp ~/.minikube/ca.crt .
+	cp ~/.minikube/ca.key .		
+	```
+	
+1. Copy and configure paddlecloud configures:
```

Comment:
Copy and update paddlecloud configurations:

Reply:
Thanks! Done.